.ci/test.sh

```diff
@@ -53,16 +53,24 @@ fi
 if [[ $TASK == "lint" ]]; then
```

Comment:
Have you tested it at Azure Pipelines? If it doesn't work there, then we should have a similar check to what we have for `check-docs` job.

Reply:
ah you're right. I haven't tested it on Azure. Since it's just linting, I don't think we need to run it more than once

I can update the condition

Reply:
updated!